bin/dev-deploy.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/sh
+
+set -e
+
+REMOTE="development"
+REMOTE_MAIN_BRANCH="master"
+CNAME_PATH="docs/CNAME"
+DRY_RUN=${1:-false}
+
+if [ -n "$(git status --porcelain)" ]; then
+  echo "Uncommitted changes detected. Please commit or stash before running this script."
+  exit 1
+else
+  echo "Working tree is clean."
+fi
+
+if ! git remote get-url "$REMOTE" >/dev/null 2>&1; then
+  echo "Remote '$REMOTE' not found. Please add it with:"
+  echo "   git remote add $REMOTE <url>"
+  exit 1
+fi
+
+if [ -f "$CNAME_PATH" ]; then
+  echo "Removing $CNAME_PATH..."
+  rm "$CNAME_PATH"
+  git add -A
```

Comment:
Optionally, we could remove the github folder too, as we don't care about workflows such as pa11y running